packages/react-composites/tests/browser/chat/hermetic/ImageOverlay.test.ts

```diff
@@ -44,7 +44,7 @@ test.describe('ImageOverlay tests', () => {
     await page.goto(
       buildUrlForChatAppUsingFakeAdapter(serverUrl, {
         localParticipant: TEST_PARTICIPANTS[1],
-        remoteParticipants: [TEST_PARTICIPANTS[0], TEST_PARTICIPANTS[2]],
+        remoteParticipants: [TEST_PARTICIPANTS[0], TEST_PARTICIPANTS[1], TEST_PARTICIPANTS[2]],
```

Comment:
Why did we add TEST_PARTICIPANTS[1] to remote participant when TEST_PARTICIPANTS[1] is already a local participant?